nautobot/extras/templates/extras/map_contact_or_team.html

```diff
@@ -0,0 +1,199 @@
+                {% render_field form.teams %}
+                {% render_field form.comments %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Contact Association</strong></div>
+            <div class="panel-body">
+                {% render_field form.contact_role %}
+                {% render_field form.contact_status %}
+            </div>
+        </div>
+        {% include 'inc/extras_features_edit_form_fields.html' %}
+    </div>
+    <div id="team" class="tabcontent">
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Team</strong></div>
+            <div class="panel-body">
+                {% render_field form.team_name %}
+                {% render_field form.team_phone %}
+                {% render_field form.team_email %}
+                {% render_field form.team_address %}
+                {% render_field form.team_contacts %}
+                {% render_field form.comments %}
+            </div>
+        </div>
+        <div class="panel panel-default">
+            <div class="panel-heading"><strong>Contact Association</strong></div>
+            <div class="panel-body">
+                {% render_field form.team_role %}
+                {% render_field form.team_status %}
+            </div>
+        </div>
+        {% include 'inc/extras_features_edit_form_fields.html' %}
+    </div>
+{% endblock %}
+
+{% block buttons %}
+    <button type="submit" name="_create" class="btn btn-primary">Assign</button>
+    <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
+{% endblock %}
+
+{% block javascript %}
+<script>
+    document.getElementById("id_location").disabled=true;
+    var contact = document.getElementById("id_contact");
+    var team = document.getElementById("id_team");
+    function disable_enable_team(){
+        if (team.disabled){
+            team.disabled=false;
+        } else {
+            team.disabled=true;
+        }
+    }
+    function disable_enable_contact(){
+        if (contact.disabled){
+            contact.disabled=false;
+        } else {
+            contact.disabled=true;
+        }
+    }
+    contact.onchange = disable_enable_team;
+    team.onchange = disable_enable_contact;
+    function openTab(evt, tab_id) {
+        var i, tabcontent, tablinks;
+        tabcontent = document.getElementsByClassName("tabcontent");
+        for (i = 0; i < tabcontent.length; i++) {
+            tabcontent[i].style.display = "none";
+        }
+        tablinks = document.getElementsByClassName("tablinks");
+        for (i = 0; i < tablinks.length; i++) {
+            tablinks[i].className = tablinks[i].className.replace(" active", "");
+        }
+        document.getElementById(tab_id).style.display = "block";
+        evt.currentTarget.className += " active";
+        if (tab_id === "assign") {
+            // enable assign tab fields
```

Comment:
It might be a good idea to match the behavior of the management command as well?

- Allow the user to input a name to create a new Contact or Team if only an email and phone exists on the location, however unlikely that is
- Migrate the phone and/or email to the existing Contact or Team if the field is blank on the Contact/Team
- Don't do anything with the shipping address / physical address fields (don't migrate them to a new contact or existing contact and don't use them for similar contact/team lookup)

Reply:
per standup: Dropdown of those four actions to take.